adam/curriculum/phase1_curriculum.py

```diff
@@ -950,21 +951,19 @@ def _make_object_under_or_over_object_curriculum(
 ) -> Phase1InstanceGroup:
     object_under = standard_object("object_0")
     object_above = standard_object("object_1", required_properties=[HAS_SPACE_UNDER])
-    bird = object_variable("bird_0", BIRD)
-    object_under_bird = standard_object("object_under_bird_0")
 
     templates = [
         Phase1SituationTemplate(
             f"object-under-object",
             salient_object_variables=[object_above, object_under],
             constraining_relations=[bigger_than(object_above, object_under)],
-            asserted_always_relations=[strictly_over(object_above, object_under)],
+            asserted_always_relations=[strictly_under(object_above, object_under)],
```

Comment:
Is this written the right way? By default, I want to read this as the first argument is under the second argument.

Reply:
Fixed, thanks for catching that!